src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: $white-accent-color;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    display: flex;
+    text-align: center;
+  }
+
+  &__link {
+    color: $main-accent-color;
+    text-decoration: none;
+    position: relative;
+    font-weight: 500;
+    font-size: 12px;
+    height: 60px;
+    line-height: 60px;
+    align-items: center;
+    text-transform: uppercase;
+    margin-left: 20px;
+    transition: color 300ms ease;
+  }
+
+  &__link:hover {
+    color: $blue-accent-color;
+  }
+
+  &__link.is-active {
+    color: $blue-accent-color;
+    margin-left: 0;
+  }
+
+  &__link.is-active::after {
+    content: '';
+    display: flex;
+    position: absolute;
+    width: 100%;
+    left: 0;
+    bottom: 0;
+    height: 4px;
+    background-color: $blue-accent-color;
```

Comment:
The variable `$blue-accent-color` is used here, but it's not defined in this file. Ensure that `$blue-accent-color` is defined in a global SCSS file or imported into this file to avoid compilation errors.